community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -79,7 +79,7 @@ public ReadOperations readOperations()
         if( !transaction.mode().allowsReads() )
         {
             throw new AuthorizationViolationException(
-                    String.format( "Read operations are not allowed for `%s` transactions.", transaction.mode().name() ) );
+                    String.format( "Read operations are not allowed for '%s'.", transaction.mode().name() ) );
```

Comment:
Does the browser apply Markdown to the error messages? (Just curious about the previous format)